docs/postgres.md

```diff
@@ -88,6 +88,36 @@ dropdb opencollective_test
 dropuser opencollective
 ```
 
+## Migrations
+
+When creating migrations and interacting with the database please follow the guidelines below;
```

Comment:
Why not a simple dot?

```suggestion
When creating migrations and interacting with the database please follow the guidelines below.
```

Reply:
Yeah, the dot works fine as well. I have a habit of adding semicolon when I have some content to show/illustrate below. 😃